src/webui/api/serialize/serialize_torrent.h

```diff
@@ -93,6 +93,6 @@ inline const QString KEY_TORRENT_SEEDING_TIME = u"seeding_time"_s;
 inline const QString KEY_TORRENT_AVAILABILITY = u"availability"_s;
 inline const QString KEY_TORRENT_REANNOUNCE = u"reannounce"_s;
 inline const QString KEY_TORRENT_COMMENT = u"comment"_s;
-inline const QString KEY_TORRENT_ISPRIVATE = u"is_private"_s;
+inline const QString KEY_TORRENT_PRIVATE = u"private"_s;
```

Comment:
> This change and line 114 in torrentscontroller.cpp is backwards incompatible for the WebAPI/WebUI.
> Are we sure to introduce disruptions for the downstream projects?

It was added in "info" quite recently and has not yet been released, so I do not consider it a problem to change it now as a hotfix.
But in "properties" it has been there for a long time. So we have several solutions:
1. still change it to "private" in an incompatible way,
2. add a new "private" field, leaving the old "is_private" as deprecated for compatibility,
3. do not change "properties", and use inconsistent names,
4. use "is_private" everywhere.

I would vote for 1. or 2.

Reply:
@Chocobo1 - I have renamed the patterns where possible. I can't rename all to "private", since the word "private" is reserved in C++. Is the updated version fine with you?

Reply:
>I would vote for 1. or 2.

Both are acceptable to me. I'll leave it to the PR author to decide.

Reply:
> > I would vote for 1. or 2.
> 
> Both are acceptable to me. I'll leave it to the PR author to decide.

I am not sure of the ramifications of that decision. Personally I am totally fine to merge it as it is (i.e. it's becoming "private" on the properties API as well) but I don't know how much that will break. What would be your advice?

Reply:
>@Chocobo1 - I have renamed the patterns where possible. I can't rename all to "private", since the word "private" is reserved in C++. Is the updated version fine with you?

Yes, it is alright now.

>I am not sure of the ramifications of that decision. Personally I am totally fine to merge it as it is (i.e. it's becoming "private" on the properties API as well) but I don't know how much that will break. 

[Third-party UIs](https://github.com/qbittorrent/qBittorrent/wiki/List-of-known-alternate-WebUIs) and third-party services (for example sonarr if I'm not mistaken) whom shows private torrent status will possibly stop working until they switch to the new value. And they probably need to maintain backward compatibility for previous qbt versions.

Reply:
Implemented 2. for maintaining backward compatibility.